modules/tensor_mechanics/test/tests/shell/static/abaqus_plate_bending.i

```diff
@@ -0,0 +1,208 @@
+# M_y = -10.0 Nm at vertices A and D (left boundary)
+
+# The disp_z at vertex C is -12.54 m using S4 elements in Abaqus.
+# The solution obtained using Moose is -12.519 m with a relative error
+# of 0.16%.
+
+[Mesh]
+  [./gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 1
+    ny = 1
+    xmin = 0.0
+    xmax = 40.0
+    ymin = 0.0
+    ymax = 20.0
+  [../]
+
+  [./c_node]
+    type = ExtraNodesetGenerator
+    input = gmg
+    new_boundary = 100
+    coord = '40.0 20.0'
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[BCs]
+  [./simply_support_x]
+    type = PresetBC
+    variable = disp_x
+    boundary = 'right top bottom left'
+    value = 0.0
+  [../]
+  [./simply_support_y]
+    type = PresetBC
+    variable = disp_y
+    boundary = 'right top bottom left'
+    value = 0.0
+  [../]
+  [./simply_support_z]
+    type = PresetBC
+    variable = disp_z
+    boundary = 'bottom left'
+    value = 0.0
+  [../]
+[]
+
+[NodalKernels]
+  [./force_C]
+    type = ConstantRate
+    variable = disp_z
+    boundary = 100
+    rate = -2.0
+  [../]
+  [./Mx_AB]
+    type = ConstantRate
+    variable = rot_x
+    boundary = bottom
+    rate = 20.0
+  [../]
+  [./Mx_CD]
+    type = ConstantRate
+    variable = rot_x
+    boundary = top
+    rate = -20.0
+  [../]
+  [./My_BC]
+    type = ConstantRate
+    variable = rot_y
+    boundary = right
+    rate = 10.0
+  [../]
+  [./My_AD]
+    type = ConstantRate
+    variable = rot_y
+    boundary = left
+    rate = -10.0
+  [../]
+[]
+
+[Preconditioning]
+  [./smp]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = NEWTON
+  line_search = 'none'
+  #nl_max_its = 2
+  nl_rel_tol = 1e-10
+  nl_abs_tol = 6e-6
+
+  dt = 1.0
+  dtmin = 1.0
+  end_time = 3
+[]
+
+[Kernels]
+  [./solid_disp_x]
+    type = ADStressDivergenceShell
+    block = '0'
+    component = 0
+    variable = disp_x
+    order = SECOND
+  [../]
+  [./solid_disp_y]
+    type = ADStressDivergenceShell
+    block = '0'
+    component = 1
+    variable = disp_y
+    order = SECOND
+  [../]
+  [./solid_disp_z]
+    type = ADStressDivergenceShell
+    block = '0'
+    component = 2
+    variable = disp_z
+    order = SECOND
+  [../]
+  [./solid_rot_x]
+    type = ADStressDivergenceShell
+    block = '0'
+    component = 3
+    variable = rot_x
+    order = SECOND
+  [../]
+  [./solid_rot_y]
+    type = ADStressDivergenceShell
+    block = '0'
+    component = 4
+    variable = rot_y
+    order = SECOND
+  [../]
+[]
+
+[Materials]
+  [./elasticity]
+    type = ADComputeIsotropicElasticityTensorShell
+    youngs_modulus = 1e3
+    poissons_ratio = 0.3
+    block = 0
+    order = SECOND
```

Comment:
Looking at this input, it seems to me like we need a different name to define the integration order for the out-of-plane integration to avoid confusion. Maybe 'thickness_order'? 'through_thickness_order'?

Reply:
I have changed 'order' to 'through_thickness_order'.